apps/playground/src/app/components/ag-grid/readonly-grid/readonly-grid.component.ts

```diff
@@ -133,7 +133,6 @@ export class ReadonlyGridComponent implements OnInit {
   public onGridReady(gridReadyEvent: GridReadyEvent): void {
     this.gridApi = gridReadyEvent.api;
     this.gridApi.sizeColumnsToFit();
-    this.gridApi.resetRowHeights();
```

Comment:
This is unnecessary when using autoheight (which we use by default) and throws a console warning.